src/Workspaces/Core/Portable/Utilities/ReferenceCountedDisposable.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Roslyn.Utilities
+{
+    /// <summary>
```

Comment:
:bulb: Can you amend this commit to leave everything from here to the final brace indented an extra level so Git detects the rename properly? You can then add a commit afterwards to outdent it.

Note that this will only work if the commit is amended (so the detected rename occurs as one commit).